gen/calc.py

```diff
@@ -335,7 +335,7 @@ def validate_cluster_packages(cluster_packages):
 
 def calculate_no_proxy(no_proxy):
     user_proxy_config = validate_json_list(no_proxy)
-    return ",".join(['*.mesos,127.0.0.1,localhost'] + user_proxy_config)
```

Comment:
`*` must **not** appear as discussed on this PR and in the JIRA ticket. 
`*.mesos` is indeed a bug and **must** be set to `.mesos`
For further evidence see: https://dcos-community.slack.com/archives/networking/p1487961197003371